server/routerlicious/packages/services-client/src/restWrapper.ts

```diff
@@ -87,11 +87,18 @@ export class RestWrapper {
                     debug(`request to ${options.url} failed ${error ? error.message : ""}`);
                 }
 
+                if (error?.response?.status === 429 && error?.response?.data?.retryAfter > 0) {
+                    setTimeout(async () => {
+                        return this.request<T>(options, statusCode);
```

Comment:
Taking another look at this, I don't think it will perform as intended. The result here is that after a retry, this call will essentially be lost. There are some good examples of using `deferred` promises in the codebase that could accomplish what you're looking to do here. Alternatively, wrapping this in a returned Promise, then using `resolve`/`reject` rather than `return *` would be a way of handling it.